<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Navigator</class>
 <widget class="QWidget" name="Navigator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: black;
}

QPushButton
{
	font: 20px, &quot;SimHei&quot;;
    color: black;
	background-color: rgb(250, 250, 250);
	border-radius: 5px;
}



QLabel
{
	color: white;
	font: 20px, &quot;SimHei&quot;;
	border:1px solid white;
}
</string>
  </property>
  <widget class="QLabel" name="NavigatorBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>70</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_5">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_6">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_7">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_8">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_9">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_10">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="nbtn1_11">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>15</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
